src/Cake.Common.Tests/Fixtures/Tools/OctopusDeployPusherFixture.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.Collections.Generic;
+using Cake.Common.Tools.OctopusDeploy;
+using Cake.Core.IO;
+using Cake.Testing.Fixtures;
+
+namespace Cake.Common.Tests.Fixtures.Tools
+{
+    internal sealed class OctopusDeployPusherFixture : ToolFixture<PushPackageSettings>
+    {
+        public List<FilePath> Packages { get; set; }
+        public OctopusDeployPusherFixture() : base("Octo.exe")
+        {
+            Packages = new List<FilePath>
+            {
+                "MyPackage.1.0.0.zip",
+                "MyOtherPackage.1.0.1.nupkg"
+            };
+            Settings.Server = "http://octopus";
+            Settings.ApiKey = "API-12345";
+        }
+
+        protected override void RunTool()
+        {
+            var tool = new OctopusDeployPusher(FileSystem, Environment, ProcessRunner, Tools);
+            tool.PushPackage(Settings, Packages == null ? null : Packages.ToArray());
```

Comment:
Not really. I ummed and ahhed about using `params` vs the `List` and was originally planning on doing the actual alias with a `params` but then changed my mind. I'll switch it to a `List` and simplify it.